client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -32,7 +32,7 @@
                     </tr>
                     <tr>
                         <td>File contents</td>
-                        <td id="file-contents"><a :href="dataset.download_url">contents</a></td>
+                        <td id="file-contents"><a :href="prefixedUrl(dataset.download_url)">contents</a></td>
```

Comment:
Backend-generated URLs should contain the prefix already, if they do not yet I think that is a bug that should be fixed.

Reply:
(if you look through the codebase for withPrefix, those are only applied when the frontend calculates the path, as in `withPrefix(`/api/histories/${historyDatasetId}/contents/${historyDatasetId}/display?filename=${path}`);`)

Reply:
@mvdbeek I can confirm that `dataset.download_url` does not have prefix included in the URL. At least that was the case when we tested `v23.0`. Not sure if something has changed since then!